pkg/backingstore/backingstore.go

```diff
@@ -819,8 +824,8 @@ func RunReconcile(cmd *cobra.Command, args []string) {
 	}
 	backingStoreName := args[0]
 	klient := util.KubeClient()
-	intervalSec := time.Duration(3)
-	util.Panic(wait.PollImmediateInfinite(intervalSec*time.Second, func() (bool, error) {
+	intervalDuration := time.Duration(3)
```

Comment:
Why change to a less descriptive name? It is now less clear that this duration represents seconds.

Reply:
Hey Guy. I've changed the variable name since I got a lint warning from VS code, that the duration variable name should not include the specific time unit name like seconds.  Duration is converted into specific time units, seconds in this case,  by multiplication by `time.Second` scale factor... Does it make sense?

Reply:
I see, I consider this a matter of style and not a lint warning, but whatever.
So perhaps it would require to separate like this:
```
intervalSec := 3
intervalDuration := time.Second * time.Duration(intervalSec)
```
and then you have both an int representing the number of seconds which can be easily printed for us humans,
and also the duration type for the wait call.